### ISSUE TITLE
[BUG?] TR1004 · Path may not include empty segments.

### ISSUE THREAD

**Issue by elderapo** [typehero/type-route#46]
I am porting my project from  `type-route@0.3.6` to `type-route@0.5.3` and started getting this error in cases like this:

```ts
const home = defineRoute("/");

const myRouter = createRouter({
  home,
  homeSomething: home.extend("/something"),
});
```

The fix is to use `home.extend("something")` instead of `home.extend("/something")` but it feels a little bit weird. Is this a bug or intended behavior?

**Comment by bradenhs** [comment]
Indeed this is a bug. Thanks for reporting! The fix is being released now in [`0.5.4`](https://github.com/typehero/type-route/releases/tag/0.5.4). Also just curious: what's your use case for needing to extend a "/" path? Is it because the route has query/state parameters?

**Comment by elderapo** [comment]
Well, I created it before the new redirection API was introduced. It was part of the workaround for the default redirections.

Offtopic: good job on the updates. Even breaking changes were easy to port and everything is working seamlessly :+1:
